ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -513,6 +514,10 @@ + (void)updateViewController:(UIViewController *)vc
 
   auto isBackButtonCustomized = !isBackTitleBlank || config.disableBackButtonMenu;
 
+  if (@available(iOS 14.0, *)) {
+    prevItem.backButtonDisplayMode = config.backButtonDisplayMode;
+  }
```

Comment:
I think we need ifdef checks for iOS version here.
Look for ifdefs for iOS 15 examples in code.